siatest/testgroup.go

```diff
@@ -293,40 +292,46 @@ func setRenterAllowances(renters map[*TestNode]struct{}) error {
 }
 
 // synchronizationCheck makes sure that all the nodes are synced and follow the
-func synchronizationCheck(miner *TestNode, nodes map[*TestNode]struct{}) error {
-	mcg, err := miner.ConsensusGet()
+func synchronizationCheck(nodes map[*TestNode]struct{}) error {
+	// Get node with longest chain.
+	var longestChainNode *TestNode
+	var longestChain types.BlockHeight
+	for n := range nodes {
+		ncg, err := n.ConsensusGet()
+		if err != nil {
+			return err
+		}
+		if ncg.Height > longestChain {
```

Comment:
What happens if two nodes have a chain of the same length but it's a different block?

Reply:
In that case the check is going to fail.